### DISCUSSION TITLE
How to shorten the theme name?

### DISCUSSION THREAD

**Discussion by SherLock707** [fastfetch-cli/fastfetch#724] [Q&A]
My current theme name looks like this:
![image](https://github.com/fastfetch-cli/fastfetch/assets/26952545/be2918d1-fac9-4cc9-a3fe-bb03b7e3a4e6)

my jsonc:
`        {
            "type": "theme",
            "key": "  󰉼"
        },`

`"format": "{1}"`
or any other index does not give me a shorten version of the name.

How do I shorten this without manually hard-coding theme name?

**Comment by CarterLi** [comment]
It's not supported. What behavior do you want?

**Reply by SherLock707** [reply]
May be a way to only display:

`Catppuccin-Mocha-Standard-Mauve-Dark [GTK2/3]`
or
`Sweet-Dark [GTK4]`
or
`Catppuccin [GTK2/3]`
or
`Catppuccin-Mocha [GTK2/3], Sweet-Dark [GTK4`
